Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -217,6 +221,14 @@ - (WMFArticleHeaderImageGalleryViewController*)headerGallery {
     return _headerGallery;
 }
 
+
+- (WMFTableOfContentsViewController*)tableOfContentsViewController{
+    if(!_tableOfContentsViewController){
+        _tableOfContentsViewController = [[WMFTableOfContentsViewController alloc] initWithSectionList:self.article.sections delegate:self];
```

Comment:
I might be missing it, but how do yo update the ToC if another article is set on the container (e.g. as the result of a fetch or another save)?